broker-daemon/block-order-worker/index.js

```diff
@@ -557,6 +557,62 @@ class BlockOrderWorker extends EventEmitter {
     return osm
   }
 
+  /**
+   * Place orders for a block order for a given amount, breaking them up based on the maximum order size
+   * @param  {BlockOrder} blockOrder Block Order to place orders on behalf of
+   * @param  {String}     baseAmount Int64 amount, in the base currency's base units, to place orders for
+   * @return {void}
+   */
+  _placeOrders (blockOrder, baseAmount) {
+    // order params
+    const { baseSymbol, counterSymbol, quantumPrice } = blockOrder
+    const baseEngine = this.engines.get(baseSymbol)
+    const counterEngine = this.engines.get(counterSymbol)
+
+    if (!baseEngine) {
+      throw new Error(`No engine available for ${baseSymbol}`)
+    }
+    if (!counterEngine) {
+      throw new Error(`No engine available for ${counterSymbol}`)
+    }
+
+    const baseMaxPayment = baseEngine.maxPaymentSize
+    const counterMaxPayment = counterEngine.maxPaymentSize
+
+    // our max payment size settings have an implied price. We need to compare that to our actual price
+    // to see which max payment size we're going to run up against.
+    const maxPaymentSizeImpliedPrice = Big(counterMaxPayment).div(baseMaxPayment)
+    let maxBaseAmountPerOrder
+
+    // quantum price is the counter/base (both in quantum units)
+    if (Big(quantumPrice).gte(maxPaymentSizeImpliedPrice)) {
+      // counter for the block order is larger than base (relative to their max payment sizes)
+      maxBaseAmountPerOrder = Big(counterMaxPayment).div(quantumPrice).round(0)
+    } else {
+      // base for the block order is larger than counter (relative to their max payment sizes)
+      maxBaseAmountPerOrder = Big(baseMaxPayment)
+    }
+
+    let baseAmountRemaining = Big(baseAmount)
+    let orderCount = 1
+
+    // split our larger block order into individual placed orders that are each under
+    // the max payment size
+    while (baseAmountRemaining.gt(0)) {
+      this.logger.info(`Placing order #${orderCount} for BlockOrder`, { blockOrderId: blockOrder.id })
+
+      let orderBaseAmount = baseAmountRemaining
+
+      if (orderBaseAmount.gte(maxBaseAmountPerOrder)) {
```

Comment:
When we are placing orders for a particular block order, do you think it matters whether we have an even amount for orders?

Example situation: 
1. max amount per order is 200
2. we have an order placed for 500

you would have the following order scheme:
`200, 200, 100`

Instead, we could have a formula to split the current order into a particular size:
`167, 167, 166`

Is this even a thing?

Reply:
That might be an optimization to do later, but I don't think it impacts anything right now